manifests/config.pp

```diff
@@ -21,7 +21,7 @@
 
   $init_defaults = {
     'MAX_OPEN_FILES' => '65535',
-  }.merge($elasticsearch::init_defaults)
+  }.stdlib::merge($elasticsearch::init_defaults)
```

Comment:
you can add the hashes with `+`. then you don't need merge and don't need stdlib.

Reply:
#1217 and just close this PR?